ui/pages/confirmation/templates/success.js

```diff
@@ -0,0 +1,94 @@
+import { IconName, IconSize } from '../../../components/component-library';
+import {
+  FontWeight,
+  BlockSize,
+  AlignItems,
+  FlexDirection,
+  JustifyContent,
+  TypographyVariant,
+  TextAlign,
+  IconColor,
+  BackgroundColor,
+} from '../../../helpers/constants/design-system';
+import { processString } from '../util';
+
+function getValues(pendingApproval, t, actions, _history) {
```

Comment:
I appreciate there is minimal logic here, but can we add the default snapshot test for these templates and maybe even some others to verify the rendering of headers if it proves straight forward?

Reply:
Absolutely,  included some tests for error and success templates.